Sources/BraveWallet/Crypto/Stores/NFTStore.swift

```diff
@@ -62,6 +71,54 @@ public class NFTStore: ObservableObject {
     userAssetManager: self.assetManager
   )
   
+  enum NFTDisplayType: Int, CaseIterable, Identifiable {
+    case visible
+    case invisible
```

Comment:
I think `hidden` is better here as it's more descriptive, especially since we are showing the 'invisible' NFTs 🙂